__tests__/app.test.js

```diff
@@ -41,6 +41,48 @@ describe('GET /api/topics', () => {
   });
 });
 
+describe('GET /api/articles', () => {
+  test('200: Responds with an array of article objects containing correct properties', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(5);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test('200: Responds with an array sorted by date, in descending order', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy('created_at', { descending: true });
+      });
+  });
+  test('200: Returned objects do not contain article.body property', () => {
```

Comment:
As this test probably passed first time you could have asserted the lack of the property in the first test instead.

Reply:
The test did pass first time :pensive: I thought it might be valuable to write an explicit test about the absence of the body property because it was in the requirements.

I've updated test first test to assert the lack of the property - my instinct was that it felt a bit too much like writing '...and...' in the test description!